Resources/Prototypes/Entities/Objects/Weapons/Guns/SMGs/smgs.yml

```diff
@@ -201,6 +201,56 @@
     zeroVisible: true
   - type: Appearance
 
+- type: entity
+  name: Drozd
+  parent: BaseWeaponSubMachineGun
+  id: WeaponSubMachineGunDrozd
+  description: A janky, ancient looking fully automatic Heavy SMG. Uses .45 magnum ammo.
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/SMGs/drozd.rsi
```

Comment:
This just doubles the vector which we used to replace the drozd.

Reply:
The Drozd is supposed to be more of an easter egg than a weapon that is regularly used, kinda like the Decker revolver (hence I added it to the loot table). If I knew how to I could maybe add some values in the yml to make it more inaccurate or have a lower rate of fire than the Vector, if that's actually possible.